api/src/utils/delete-from-require-cache.ts

```diff
@@ -1,7 +1,13 @@
 import { createRequire } from 'node:module';
+import logger from '../logger.js';
 
 const require = createRequire(import.meta.url);
 
 export function deleteFromRequireCache(modulePath: string): void {
-	delete require.cache[require.resolve(modulePath)];
+	try {
+		const moduleCachePath = require.resolve(modulePath);
+		delete require.cache[moduleCachePath];
+	} catch (error) {
+		logger.warn(`Module cache not found for ${modulePath}, skipped cache delete.`);
+	}
```

Comment:
Couldn't this be just a simple `if` (if `require.cache[moduleCachePath]` exists)?
Also I don't think it's necessary to log this, or at least the level should be changed from `warn` to `trace`.

Reply:
The `require.resolve` is what throws the `MODILE_NOT_FOUND` error so doing that would not resolve this issue.

I can update the warn to a trace if you believe that to be more appropriate. Logging it as a warn was mentioned in the issue so that is what I used.

Reply:
Ah I see, thanks 😄👍

I don't think that's really relevant to log, and might be more confusing than helping (since you cannot do anything against it anyway). Therefore I'd go with the trace level.

Reply:
Alright, sounds good. Will update it tonight.